proect_on_desktop.css

```diff
@@ -0,0 +1,175 @@
+* {
+    box-sizing: border-box;
+    margin: 0;
+    padding: 0;
+}
+
+body {
+    margin: 0;
+    font-family: Montserrat;
+    background-color: black;
+    color: white;
+}
+
+ul {
+    list-style: none;
+}
+
+.a_little_down_cart {
```

Comment:
class name sounds unclear. 
```.header__cart_icon``` seems to be better. What do you think?

Reply:
so great, yes :D

Reply:
I'll change it